tardis/model/tests/data/csvy_model_to_test_abundances.yml

```diff
@@ -34,7 +34,7 @@ montecarlo:
   # --------
   #nthreads: 1
 
-  last_no_of_packets: 1.e+5
+  last_no_of_packets: 1.e+4
```

Comment:
why did we change this. AFAICT it is only to read in the model

Reply:
for actually running it we should do at least 100k packets

Reply:
The simulation is not being run, so the value doesn't matter.  But let's minimize changes and also keep realistic values for important parameters like this.